packages/apptive_grid_form/lib/src/widgets/form_widget/form_content_widgets.dart

```diff
@@ -70,6 +73,12 @@ class FormDataWidget extends StatefulWidget {
   /// Flag to hide the form description, default is false
   final bool hideDescription;
 
+  /// Style for the Form Description. If no style is provided [bodyText1] of the [TextTheme] will be used
```

Comment:
`bodyText1` is from Material Style. This should have `bodyLarge` as this is what would actually be used

Reply:
done